docs/VictoriaLogs/data-ingestion/Fluentd.md

```diff
@@ -0,0 +1,109 @@
+---
+weight: 2
+title: Fluentd setup
+disableToc: true
+menu:
+  docs:
+    parent: "victorialogs-data-ingestion"
+    weight: 2
+aliases:
+  - /VictoriaLogs/data-ingestion/Fluentd.html
+  - /victorialogs/data-ingestion/fluentd.html
+  - /victorialogs/data-ingestion/Fluentd.html
+---
+VictoriaLogs supports given below Fluentd outputs:
+- [Loki](#loki)
+- [HTTP JSON](#http)
+
+## Loki
+
+Specify [loki output](https://docs.fluentd.io/manual/pipeline/outputs/loki) section in the `fluentd.conf`
+for sending the collected logs to [VictoriaLogs](https://docs.victoriametrics.com/victorialogs/):
+
+```conf
+<match **>
+  @type loki
+  url "http://localhost:9428/insert"
+  <buffer>
+    flush_interval 10s
+    flush_at_shutdown true
+  </buffer>
+  custom_headers {"VL-Msg-Field": "log", "VL-Time-Field": "time", "VL-Stream-Fields": "path"}
+  buffer_chunk_limit 1m
+</match>
+```
+
+## HTTP
+
+Specify [http output](https://docs.fluentd.io/manual/pipeline/outputs/http) section in the `fluentd.conf`
+for sending the collected logs to [VictoriaLogs](https://docs.victoriametrics.com/victorialogs/):
+
+```fluentd
+<match **>
+  @type http
+  endpoint "http://localhost:9428/insert/jsonline"
+  headers {"VL-Msg-Field": "log", "VL-Time-Field": "time", "VL-Stream-Fields": "path"}
+</match>
+```
+
+Substitute the host (`localhost`) and port (`9428`) with the real TCP address of VictoriaLogs.
+
+See [these docs](https://docs.victoriametrics.com/victorialogs/data-ingestion/#http-parameters) for details on the query args specified in the `uri`.
+
+It is recommended verifying whether the initial setup generates the needed [log fields](https://docs.victoriametrics.com/victorialogs/keyconcepts/#data-model)
+and uses the correct [stream fields](https://docs.victoriametrics.com/victorialogs/keyconcepts/#stream-fields).
+This can be done by specifying `debug` [parameter](https://docs.victoriametrics.com/victorialogs/data-ingestion/#http-parameters) in the `uri`
```

Comment:
```suggestion
This can be done by specifying `debug` [parameter](https://docs.victoriametrics.com/victorialogs/data-ingestion/#http-parameters) in the `endpoint`
```